packages/fluxcd-source-controller/0.24.4.1/package.yaml

```diff
@@ -5,7 +5,8 @@ metadata:
 spec:
   refName: fluxcd-source-controller.community.tanzu.vmware.com
   version: 0.24.4.1
-  releasedAt: "2022-05-03T11:14:08Z"
+  releaseNotes: "Fixing valuesSchema bug in release 0.24.4"
```

Comment:
nit: Fixed valuesSchema bug and updated base image to ....

Reply:
done